src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -25,7 +25,7 @@
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
 
     private final Person toAdd;
-
+    private final boolean isMutating = true;
```

Comment:
It should be fine to directly return the `boolean` value from the method, no need to store it in a variable.